src/CommunityToolkit.Maui.MediaElement/Services/BoundServiceBinder.android.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Android.OS;
+using CommunityToolkit.Maui.Media.Services;
+
+namespace CommunityToolkit.Maui.Services;
+
+class BoundServiceBinder(MediaControlsService mediaControlsService) : Binder
+{
+	public MediaControlsService? Service { get; private set; } = mediaControlsService;
```

Comment:
If we want this to be immutable, please remove the `private set`.

Please, review all properties that have `private set` and delete those if immutability is desired

Reply:
I made the bound service immutable. I hope that resolves this issue. I also removed private set and it only has a getter now.